setup.py

```diff
@@ -44,7 +44,7 @@
 DOWNLOAD_URL = 'https://github.com/rupertford/melody'
 DESCRIPTION = 'lightweight python parameter search tool'
 LONG_DESCRIPTION = open('README.md').read()
-LICENSE='OSI Approved :: BSD 3-Clause License'
+LICENSE = 'OSI Approved :: BSD 3-Clause License'
```

Comment:
Yes, I thought I'd fix any minor formatting errors here rather than create a new ticket. I forgot to mention that (apart from in the commit notes). Apologies.